app/policies/mfa_policy.rb

```diff
@@ -15,10 +15,16 @@ def multiple_factors_enabled?
     mfa_user.enabled_mfa_methods_count > 1
   end
 
-  def three_or_more_factors_enabled?
+  def more_than_two_factors_enabled?
     mfa_user.enabled_mfa_methods_count > 2
   end
 
+  def sufficient_factors_enabled?
+    mfa_user.enabled_mfa_methods_count > 1 ||
+      (FeatureManagement.backup_codes_2fa? &&
```

Comment:
One suggestion I'd have is to make the name of the flag a bit more descriptive. Something like `allow_backup_codes_only_enabled?` or something like that. That way we won't have to refer to the code when decided how to toggle it.

Reply:
@stevegsa the 2nd screen is added by a different sub-task and PR